tasks/methodology/word-embeddings/conda.yaml

```diff
@@ -0,0 +1,10 @@
+name: word_embedding
+channels:
+  - defaults
+dependencies:
+  - python=3.7
+  - pandas=0.24.2
+  - numpy=1.16.4
+  - gensim=3.8.0
```

Comment:
move gensim into pip dependency as it's not present in conda repository
gensim=3.8.0 should be changed to gensim==3.8.0

Reply:
I have resolved the comment and checked in the updated code.